contrib/init/systemd/docker.service

```diff
@@ -9,7 +9,7 @@ Type=notify
 # the default is not to use systemd for cgroups because the delegate issues still
 # exists and systemd currently does not support the cgroup feature set required
 # for containers run by docker
-ExecStart=/usr/bin/dockerd -H fd://
+ExecStart=/usr/bin/dockerd
```

Comment:
Except, my docker instance works just fine without that option.  That option also directly conflicts with daemon.json.  It's not possible to configure the `hosts` field in `daemon.json` so long as that option exists.